cmd/controller/main.go

```diff
@@ -65,7 +69,7 @@ func main() {
 		PRImage:                  *prImage,
 		ImageDigestExporterImage: *imageDigestExporterImage,
 	}
-	sharedmain.Main(ControllerLogKey,
+	sharedmain.MainWithContext(injection.WithNamespaceScope(context.TODO(), *namespace), ControllerLogKey,
```

Comment:
Shouldn't it be `context.Background()` instead ? or even better `signals.NewContext()` (https://github.com/knative/pkg/blob/master/injection/sharedmain/main.go#L122)

Reply:
fixed